cogboard-webapp/src/components/WidgetForm/index.js

```diff
@@ -90,7 +90,7 @@ const WidgetForm = ({ renderActions, ...initialFormValues }) => {
                 shrink: true
               }}
               inputProps={{
-                min: ROWS_MIN,
+                min: WIDGET_ROWS_MIN,
                 max: 4,
```

Comment:
If we have constant for min, we should have for max if it is not dynamic.

Reply:
done